### ISSUE TITLE
Table creation doesn't work

### ISSUE THREAD

**Issue by Deepfreeze10** [Happening/Text#2]
The test example makes a table. But the markup doesn't show a table anymore.
Also the text styling elements don't work when using it in the table.

**Comment by EmielM** [comment]
Good catch, we've moved to a more generic implementation for markdown that also works in the Wall/Blog/comments all over Happening. I think this broke support for tables, will add it to our backlog.
